v2/azure-arm.yaml

```diff
@@ -1308,6 +1308,51 @@ objectModelConfiguration:
         $generatedConfigs:
           ConnectionString: $.Status.ConnectionString
           InstrumentationKey: $.Status.InstrumentationKey
+    2023-01-01:
+      ActionGroup:
+        $export: true
+        $supportedFrom: v2.4.0
+      AutomationRunbookReceiver:
+        WebhookResourceId:
+          $armReference: true
+      AzureFunctionReceiver:
+        FunctionAppResourceId:
+          $armReference: true
+      LogicAppReceiver:
+        ResourceId:
+          $armReference: true
+    2022-10-01:
+      AutoscaleSetting:
+        $export: true
+        $supportedFrom: v2.4.0
+        TargetResourceUri:
+          $armReference: true
+      MetricTrigger:
+        MetricResourceUri:
+          $armReference: true
+    2018-03-01:
+      MetricAlert:
+        $export: true
+        $supportedFrom: v2.4.0
+      WebtestLocationAvailabilityCriteria:
+        ComponentId:
+          $armReference: true
+      MetricAlertProperties:
+        Scopes:
+          $armReference: true
+    2023-03-15-preview:
```

Comment:
There's a 2022-06-15 stable API for these resources... wondering if we should prefer that over the preview?

Reply:
Done